app/views/crime_applications/index.html.erb

```diff
@@ -39,7 +39,7 @@
               LAA-a1234b
             </td>
             <td class="govuk-table__cell">
-              <strong class="govuk-tag govuk-tag--blue">In progress</strong>
+              <strong class="govuk-tag govuk-tag--blue"><%= app.pretty_status %></strong>
```

Comment:
The color of the tag also changes if it is a submitted application. Not that we have submitted anyways but we could as well leave it prepared or we will forget.